### PR TITLE
[#11547] Remove outdated instructor help information

### LINKED ISSUE #11547: Outdated question in instructor help page
This section of instructor help page seems to be outdated.

Screen Shot 2022-01-31 at 11 52 09 PM

Looking at the students page, there doesn't seem to be a way to filter out students and the "Copy Emails" button doesn't seem to exist anywhere else in the repo.

Screen Shot 2022-01-31 at 11 56 59 PM

**Possible Change**
If single student: 
Use search feature to find student or find student in students page.  

If multiple students:
Under the courses page, click on other actions ->  view. Click download student list, copy emails from there.

### PR DESCRIPTION
**Pull request by fsgmhoward**
Fixes #11547

**Outline of Solution**

Instructor help page for _How do I email a student or students in my course?_ section has been deleted as it is no longer relevant. Please see the outcome at this comment: https://github.com/TEAMMATES/teammates/pull/11555#issuecomment-1056334359

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html [hunk 1]
```diff
@@ -147,18 +147,50 @@ <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
...
       </li>
       <li>
-        Click
-        <button class="btn btn-sm btn-secondary">Copy Emails</button> to copy their email addresses to your clipboard.
+        Use mouse to select students' email addresses, and copy them to your clipboard.
       </li>
       <li>
         Use your preferred email provider to email the students.
       </li>
     </ol>
```
Comment (samuelfangjw) [inline src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:161]: I think we could be clearer with the word "filter". Filter might refer to the old students page where there was probably an actual option to filter the list of students, at least that's what the word leads me to believe. Debatable though. How about 

```
1. On the <b>Students</b> page, select the students' email addresses and copy them to your clipboard.
2. Use your preferred email provider to email the students.
```

What do you think?

Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:161]: Thanks for the comment. Sounds good for me.

I also thought for a while when I edited this part. I left it there as I felt it can also mean using the sorting headers to find the student desired. However, what you mention is a valid concern and I will edit it accordingly.

Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html]: This has been fixed as per your comment.

Comment (samuelfangjw) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html]: There's an added [comment](https://github.com/TEAMMATES/teammates/issues/11547#issuecomment-1030882558) on the original issue. Should we rephrase it to something along the lines of "how can i get the email addresses of my students"?

Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html]: Ok I guess we can put this on hold first until the original issue has a conclusion.

File: src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html [hunk 2]
```diff
@@ -142,26 +140,8 @@ <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
       <tm-student-result-table [studentTables]="exampleMultipleStudentResultTables" [isActionButtonsEnabled]="false"></tm-student-result-table>
     </tm-example-box>
   </tm-instructor-help-panel>
-  <!--Question-->
-  <tm-instructor-help-panel class="instr-help-qn-last" #question [hidden]="key && !showQuestion.includes(StudentsSectionQuestions.STUDENT_EMAIL)"
-                            [id]="StudentsSectionQuestions.STUDENT_EMAIL"
```
Comment (samuelfangjw) [inline src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:147]: Hey Howard, I believe we should be able to safely remove this constant as well, could you help to check?

Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:147]: You made a point. It has been removed in the last commit. Thanks for the comment.

Comment (samuelfangjw) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:147]: Let's remove the enum const here as well, I don't believe it's being used anywhere else.

https://github.com/TEAMMATES/teammates/blob/9cc2e8451096b900b405d8c698291adf0e782e09/src/web/app/pages-help/instructor-help-page/instructor-help-students-section/students-section-questions.ts#L25-L28

Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:147]: This is updated in the latest commit. One other missing part is also fixed.

File: src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html [hunk 3]
```diff
@@ -96,8 +96,6 @@ <h4 *ngIf="displaySubsection(0, 3)">Student Records</h4>
       </li>
     </ol>
   </tm-instructor-help-panel>
-  <!-- Section -->
-  <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
   <!--Question -->
   <tm-instructor-help-panel class="instr-help-qn-first" #question [hidden]="key && !showQuestion.includes(StudentsSectionQuestions.STUDENT_SEARCH)"
```
Comment (samuelfangjw) [inline src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html:100]: One last thing, the first and last questions of each section have the class `instr-help-qn-first` and `instr-help-qn-last` respectively which applies some padding. Since we removed one section and one qns, need to ensure this class is applied correctly to all the questions in the two different sections.

**Review by samuelfangjw — CHANGES_REQUESTED** [review]

File: src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html (hunk 3 above)
Comment (fsgmhoward) [reply src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html]: Thank you for pointing so many points I missed out. I have updated this in the last commit.

**Review by samuelfangjw — APPROVED** [review]
LGTM!

**Review by jianhandev — APPROVED** [review]
LGTM, remember to update the PR description too!

**Comment by fsgmhoward** [comment]
@jianhandev I have updated the description.
